README.md

```diff
@@ -3,9 +3,7 @@
 ## Learning Goals
 
 - Identify the three essential pillars of front-end web programming
-- Cause a change to given code so that DOM updating effect is seen
-- Cause a change to given code so that server-side behavior is stubbed in
-- Cause a change to given code so that event listening has an effect
+- Walk through implementing the three pillars in a simple web application
```

Comment:
Identify steps to implement the Three Essential Pillars of Front-End Web programming in a simple web application.